api_reference/source/index.md

```diff
@@ -9,14 +9,16 @@ html_theme.sidebar_secondary.remove: true
 
 # Arize Phoenix API Reference
 
-Welcome to Phoenix's API reference.
+Welcome to Arize Phoenix's API reference. This API reference details the packages, modules, and functions included in Phoenix's API and how to use them. To get a complete guide on how to use Phoenix, including tutorials, quickstarts, and concept explanations, see the [complete documentation](https://docs.arize.com/phoenix)
 
 ## API Definition
 
 ```{toctree}
 :maxdepth: 2
 
 api/inferences
-api/session
+api/client_package
+api/db
```

Comment:
We can exclude DB-related modules. These are implementation details that users don't need to know about.